scripts/us_census/pep/pep_by_srh/process.py

```diff
@@ -865,11 +1234,19 @@ def _consolidate_county_files():
             lambda r: _calculate_asis_measure_method(r.YEAR, r.SV), axis=1)
 
         if file == county_file[0]:
+            #added by Shamim to keep last values
```

Comment:
Pls explain the reason for drop_duplicates() here and below.

Reply:
Sure, will need some time to analyze this. I'll keep you posted.

Reply:
After drop_duplicates(), got some point deletions for eight places under 2020 year in differ(similar to Harsha's case). As I have considered the latest '2020' year data.